.github/workflows/ci.yaml

```diff
@@ -0,0 +1,34 @@
+name: CI
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: fedora:35
+    steps:
+      - name: Install libfaketime
+        run: dnf install -y faketime
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+
+      - name: Install pipenv
+        run: python -m pip install --upgrade pipenv
```

Comment:
You could just install this from the fedora repo at the same time as installing faketime